workspace-manager-clienttests/src/main/resources/suites/BasicIntegration.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "BasicIntegration",
   "description": "Integration tests.",
-  "serverSpecificationFile": "workspace-dev.json",
+  "serverSpecificationFile": "workspace-local.json",
```

Comment:
This change (and the similar ones in BasicAuthenticated, BasicUnauthenticated) will affect any WSM developers who run these manually. Is that what you intended? If so, I think it should be noted in the PR description so everyone has a heads up.

For clarity, I think we should set the env var in the GH action script to specify which server we are testing against. This will be particularly helpful once we switch over to testing against dev -- then it's just a change to the env var. (https://github.com/DataBiosphere/terra-test-runner#override-the-server-from-the-command-line)

Reply:
Added a note in GHA on how to override server specification with env var.